helix-core/src/test/java/org/apache/helix/manager/zk/TestZkBaseDataAccessor.java

```diff
@@ -50,11 +66,6 @@ public void afterMethod() {
     }
   }
 
-  @AfterClass
-  public void after() {
-    int a =1;
-  }
```

Comment:
Could we make sure we have an AfterClass where we close all outstanding/existing ZkClient connections? This is very important for the stability of our tests.

Reply:
done.